web-client/public/js/update-app.js

```diff
@@ -372,8 +372,9 @@ const enableNodeColoringUI = function () {
 const loadExpressionDatabase = function (isTopDataset) {
     startLoadingIcon();
     queryExpressionDatabase({
+        type:"ExpressionData",
         dataset: isTopDataset ? grnState.nodeColoring.topDataset : grnState.nodeColoring.bottomDataset,
-        genes : grnState.workbook.genes
+        genes : grnState.workbook.genes.map(x => {return x.name;}).join(",")
```

Comment:
That sneaky `x` again 😅—here I think the ideal name is clear, yes? It’s mapping from a `genes` array, so each member item is a…